index.html

```diff
@@ -72,6 +72,21 @@
       href="assets/img/CALCULATOR.png"
       alt="CALCULATOR"
     />
+    <nav class="navbar">
+      <div class="brand-title">CALCULATOR | YOSHITHA</div>
```

Comment:
Don't change the font style of the main heading @Mayureshkaulwar

Reply:
Ok

Reply:
Can I create new pull request?

Reply:
Sure @Mayureshkaulwar

Reply:
I have created new pull request, please merge that